Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java

```diff
@@ -82,10 +83,16 @@ private void initBinder(WebDataBinder binder) {
      *         page
      */
     @RequestMapping(value = "auth/conceptlist/addconceptwrapper")
-    public String prepareConceptWrapperAdd(@ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean,
-            HttpServletRequest req, ModelMap model) {
+    public String prepareConceptWrapperAdd(@RequestParam(value = "wrapperId", required = false) String wrapperId, @ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean,
```

Comment:
just create the ConceptWrapperAddBean, don't inject it. since it's just a helper class, instantiating would be fine.

Reply:
Changed. Just instantiated ConceptWrapperAddBean.